contracts/base/Dispatcher.sol

```diff
@@ -291,13 +291,13 @@ abstract contract Dispatcher is Payments, V2SwapRouter, V3SwapRouter, V4SwapRout
 
     /// @notice Function to be used instead of msg.sender, as the contract performs self-reentrancy and at
     /// times msg.sender == address(this). Instead _msgSender() returns the initiator of the lock
-    /// @dev overrides BaseActionsRouter._msgSender in V4Router
-    function _msgSender() internal view override returns (address) {
+    function _msgSender() internal view returns (address) {
```

Comment:
i think we should remove this now and just use `msgSender` although gas will go up a little

Reply:
kk

Reply:
didn't make gas go up at all - i think u had this in a separate pr where we were discussing the same thing